CBIIT/Cancer-Center-Smoking-Cessation-Toolkit
# NCIDCCPS-2146: Adding virtual icon for events page

To test on LOCAL:

1. /admin/structure/taxonomy/manage/event_types/overview
Add "Virtual Event"

2. Modify an "Event" that appears on the DCCPS Supported Events page: /events
Change "Event Type" to Virtual Event. Save.

3. Refresh Supported Events page and confirm that new category and icon show up where expected.
